### PR TITLE
Update for new PyPI URL

### PR DESCRIPTION
**Pull request by wesm**
Well, this is super fun. @jakirkham do we know what happened here / is there some more reliable way to get to the download URL?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ package:
   version: "0.13.7"
 
 source:
-  fn: impyla-0.13.7.tar.gz
-  url: https://pypi.python.org/packages/source/i/impyla/impyla-0.13.7.tar.gz
-  md5: 331f581a45d48b0482b62d5c67005f8e
+  fn: impyla-0.13.8.tar.gz
+  url: https://pypi.python.org/packages/36/35/bcf5ac18e3689948e980e2a0b33d48f46cb8569e1d0f2b17ea7147e88356/impyla-0.13.8.tar.gz
```
Comment (mariusvniekerk) [inline recipe/meta.yaml:7]: Wow

**Comment by jakirkham** [comment]
Basically use pypi.io instead. See this issue for details ( https://github.com/conda-forge/staged-recipes/issues/481 ).

**Comment by wesm** [comment]
Seems like `pypi.io` has weird URLs too

**Comment by jakirkham** [comment]
Should be. Just change `pypi.python.org` to `pypi.io`. Here is the URL for this ( https://pypi.io/packages/source/i/impyla/impyla-0.13.7.tar.gz ). It redirects to the `pypi.python.org` URL, but it is less funky.

**Comment by jakirkham** [comment]
Shouldn't be. Just change `pypi.python.org` to `pypi.io`. Here is the URL for this ( https://pypi.io/packages/source/i/impyla/impyla-0.13.7.tar.gz ). It redirects to the `pypi.python.org` URL, but it is less funky.

**Comment by wesm** [comment]
@jakirkham indeed you're right. it would be cool if pypi.io had some explicit indication of a "permalink" but this works for me =)

**Comment by jakirkham** [comment]
Yeah, that would be nice. It would also be nice if the normal `pypi.python.org` redirected to these new hashed URLs, but no such luck. Basically this is as good as we are going to get. See the history in this PyPI [issue](https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package).

**Pull request merged**